lib/events/codes.go

```diff
@@ -294,6 +294,14 @@ const (
 	MFADeviceAddEventCode = "T1006I"
 	// MFADeviceDeleteEventCode is an event code for users deleting MFA devices.
 	MFADeviceDeleteEventCode = "T1007I"
+	// BillingCardCreateCode is emitted when a user creates a new credit card.
+	BillingCardCreateCode = "T1008I"
```

Comment:
If you look at numbers, the `T10xxx` is used for user login events. The billing deserve own group. 

Lets use `TBL` prefix for that:

`TBL01I`
`TBL02I`

Reply:
revised